web_widget_x2many_2d_matrix/static/src/components/x2many_2d_matrix_renderer/x2many_2d_matrix_renderer.esm.js

```diff
@@ -162,6 +164,12 @@ export class X2Many2DMatrixRenderer extends Component {
             record: record,
             name: this.matrixFields.value,
         };
+        const domain = record.fields[this.matrixFields.value].domain;
```

Comment:
Missing the check for null `record` (there is a check in line 160)

`const domain = record ? record.fields[this.matrixFields.value].domain : null;`

Reply:
I just check again, it seems that there is always record.